loadtest/control/simulcontroller/scheduled_posts.go

```diff
@@ -0,0 +1,155 @@
+package simulcontroller
+
+import (
+	"fmt"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/user"
+	"github.com/mattermost/mattermost/server/public/model"
+	"math/rand"
+	"time"
+)
+
+func (c *SimulController) createScheduledPost(u user.User) control.UserActionResponse {
+	if ok, resp := control.ScheduledPostsEnabled(u); resp.Err != nil {
+		return resp
+	} else if !ok {
+		return control.UserActionResponse{Info: "scheduled posts not enabled"}
+	}
+
+	channel, err := u.Store().CurrentChannel()
+	if err != nil {
+		return control.UserActionResponse{Err: control.NewUserError(err)}
+	}
+
+	var rootId = ""
+	if rand.Float64() < 0.25 {
+		post, err := u.Store().RandomPostForChannel(channel.Id)
+		if err == nil {
+			if post.RootId != "" {
+				rootId = post.RootId
+			} else {
+				rootId = post.Id
+			}
+		}
+	}
+
+	if err := sendTypingEventIfEnabled(u, channel.Id); err != nil {
+		return control.UserActionResponse{Err: control.NewUserError(err)}
+	}
+
+	message, err := createMessage(u, channel, false)
```

Comment:
I assume we can schedule replies as well. A case to consider adding.

Reply:
Done, but this doesn't matter as all thats happening from isReply param is changing the avg word length, nothing else The part that send scheduled post as a reply is above where I set the rootId. But fixed this anyways as it is now semantically correct.